src/App.jsx

```diff
@@ -1,11 +1,11 @@
 // React Imports
 import React from 'react';
 import { BrowserRouter } from 'react-router-dom';
-import { SessionProvider } from '@contexts';
 // Material UI Imports
 import { ThemeProvider } from '@mui/material/styles';
 import CssBaseline from '@mui/material/CssBaseline';
 // Context Imports
+import { SessionProvider } from '@contexts';
 import UserDataContextProvider from './contexts/UserDataContext';
```

Comment:
might as well make this one `@contexts` too

Reply:
`UserDataContext` is not used as an export from `contexts/index.js` to avoid dependency cycles discovered by the linter. It's likely complaining for importing from index.js from one of the sibling contexts when you could simply import the sibling directly.

<img width="651" alt="Screen Shot 2023-08-01 at 15 32 47" src="https://github.com/codeforpdx/PASS/assets/14917816/e7ff1c11-54e4-4184-8303-ab1a38308a68">

But this lint error vanishes if we import the context explicitly as shown here.